Mathlib/Topology/Algebra/InfiniteSum/Group.lean

```diff
@@ -397,3 +397,29 @@ theorem tprod_const [T2Space G] (a : G) : ∏' _ : β, a = a ^ (Nat.card β) :=
       simpa [multipliable_const_iff] using ha
 
 end IsTopologicalGroup
+
+section CommGroupWithZero
+variable {K : Type*} [CommGroupWithZero K] [TopologicalSpace K] [ContinuousMul K]
+
+open Finset in
+lemma Multipliable.congr_cofinite'
```

Comment:
```suggestion
/--
See also `Multipliable.congr_cofinite`, which instead requires the target to be a group under multiplication (and hence fails for infinite products in a ring.
-/
lemma Multipliable.congr_cofinite'
```
(or something analogous: I think your comment in the PR description is good and should be included as documentation)

Reply:
Sure. I can also add a comment to the doc of `Multipliable.congr_cofinite` pointing the other way.